### ISSUE TITLE
SSR fetch doesn't work for external requests

### ISSUE THREAD

**Issue by koffeinfrei** [sveltejs/kit#2028]
### Describe the bug

Using `fetch` in SSR treats requests as relative with the default config value of "" (empty string) for `options.paths.base`, instead of as an external request.

The reason is the statement https://github.com/sveltejs/kit/blob/2a1e9795ad9773e04669a7b9d0234caa0b688ade/packages/kit/src/runtime/server/page/load_node.js#L113 that always resolves to `true` with the default value.

### Reproduction

```javascript
<script context="module">
  import { fetchMovie } from '../../fetchMovie';

  export async function load({ page, session, fetch }) {
      const result = await fetch('https://duckduckgo.com/');
     // --> result is not from duckduckgo, but from the svelte-kit app
  }
</script>
```

### System Info

```shell
npx: installed 1 in 0.935s

  System:
    OS: Linux 5.11 Ubuntu 21.04 (Hirsute Hippo)
    CPU: (4) x64 Intel(R) Core(TM) i7-7600U CPU @ 2.80GHz
    Memory: 963.18 MB / 15.38 GB
    Container: Yes
    Shell: 5.1.4 - /bin/bash
  Binaries:
    Node: 12.20.0 - ~/.nvm/versions/node/v12.20.0/bin/node
    Yarn: 1.12.3 - /usr/bin/yarn
    npm: 6.14.8 - ~/.nvm/versions/node/v12.20.0/bin/npm
  Browsers:
    Chromium: 92.0.4515.107
    Firefox: 90.0
```

### Severity

blocking all usage of SvelteKit

**Comment by benmccann** [comment]
Are you using the latest version of SvelteKit? This should have been fixed in https://github.com/sveltejs/kit/pull/1980

`https://duckduckgo.com/` doesn't start with `/` so I don't see from the code how it would fail

**Comment by koffeinfrei** [comment]
> Are you using the latest version of SvelteKit? This should have been fixed in #1980

Right, this has been fixed right between me detecting the bug and reporting it. Sorry for the noise.
